### PR TITLE
Update avatica-core to 1.22.0

### PR DESCRIPTION
**Pull request by SujanSanjula96**
Relates to https://github.com/wso2/product-is/issues/14585

### CODE REVIEW

File: solr/8.11.1.wso2v6/pom.xml [hunk 1]
```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```
Comment (piraveena) [inline solr/8.11.1.wso2v6/pom.xml:4]: change the license to LLC

**Review by piraveena — APPROVED** [review]

File: solr/8.11.1.wso2v6/pom.xml (hunk 1 above)
Comment (SujanSanjula96) [reply solr/8.11.1.wso2v6/pom.xml]: Fixed it to LLC

File: solr/8.11.1.wso2v6/pom.xml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+                            calcite-linq4j;scope=compile|runtime;inline=false,
+                            calcite-core;scope=compile|runtime;inline=false,
+                            avatica-core;scope=compile|runtime;inline=false,
+                            commons-collections4;scope=compile|runtime;inline=false,
+                            caffeine;scope=compile|runtime;inline=false,
+                            zookeeper;scope=compile|runtime;inline=false,
+                            metrics-core;scope=compile|runtime;inline=false,
+                        </Embed-Dependency>
+                        <Include-Resource>@solr-core-${version.solr}.jar!/*</Include-Resource>
+                        <!--Dynamic imports enabled here because solr uses dynamic class loading to import lucene -->
```
Comment (sahandilshan) [inline solr/8.11.1.wso2v6/pom.xml:224]: Add a space before starting the comment, and finish the comment with a full stop.

Comment (SujanSanjula96) [reply solr/8.11.1.wso2v6/pom.xml:224]: Fixed in https://github.com/wso2/orbit/pull/837

**Pull request merged**